bin/merge.py

```diff
@@ -3,43 +3,101 @@
 
 import json
 from sys import argv
+from pathlib import Path
 
 
-def merge(filenames, filecount):
-    merged = {
-        "symbols": {},
-        "bigrams": {},
-        "trigrams": {},
-    }
+# sort the merged dictionary by symbol frequency (requires CPython 3.6+)
+def _sort_ngram_by_frequency(table, precision=3):
+    sorted_dict = {}
+    for key, count in sorted(table.items(), key=lambda x: -x[1]):
+        freq = round(count, precision)
+        if freq > 0:
+            sorted_dict[key] = freq
+    return sorted_dict
 
-    # merge dictionaries
+
+def sort_by_frequency(corpus: dict, precision=3):
+    for ngram in range(1, len(corpus["freq"].keys())+1):
+        ngram = str(ngram)
```

Comment:
Again: `ngram` is either an integer or a string. I know Python allows it but we really have to move away from this bad practice.